lua/weapons/weapon_gdo.lua

```diff
@@ -406,6 +444,9 @@ end
 
 function SWEP:OnRemove()
 	self:Holster()
+	if self.gate then
```

Comment:
this will have side effect as triggering transmit even if there is iris computer, so i think this is not necessary to clear input at all, just commend this code for now.

Reply:
There was a very specific reason for this in which I completely forgot about. I will comment that out.